recipes/htseq/0.6.1/meta.yaml

```diff
@@ -12,6 +12,8 @@ package:
   version: 0.6.1
 
 requirements:
+  build:
+  - {{ compiler('c') }}
```

Comment:
Is this required/recommended? I added it in an attempt to get `circleci build` tests to pass, but they never did. Not sure if I should keep it or not.

Reply:
yes it is, if a compiler is needed :)